src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -227,7 +227,7 @@ public String getBuilderId() {
   @Override
   @NotNull
   public ModuleType getModuleType() {
-    return FlutterModuleUtils.getModuleTypeForFlutter();
+    return ModuleTypeManager.getInstance().findByID(FlutterModuleUtils.getModuleTypeIDForFlutter());
```

Comment:
I was avoiding creating instances of ModuleTypes were I could, but don't have a strong preference here -